go.mod

```diff
@@ -78,3 +78,5 @@ require (
 go 1.13
 
 replace github.com/pingcap/check => github.com/tiancaiamao/check v0.0.0-20191119042138-8e73d07b629d
+
+replace github.com/pingcap/tipb => github.com/crazycs520/tipb v0.0.0-20191126095309-9e2a1f8d203b
```

Comment:
Have you file a PR to `tipb`?

Reply:
Ye, This PR description already said: https://github.com/pingcap/tipb/pull/152

Reply:
I think we don't have to add a new executor `MemTableScan` because we can reuse `TableScan` executor.

Reply:
We don’t need the new version of `tipb` anymore.